Dockerfile

```diff
@@ -1,29 +1,52 @@
 ARG node_version=20.13-alpine3.19
 
-FROM node:${node_version} as builder
+FROM --platform=$BUILDPLATFORM node:${node_version} as build
```

Comment:
This means we run yarn at native performance, then we transfer the cache and re-run on the target platform. Because we already have most of the needed packages in the yarn cache, it speeds up an emulated yarn install significantly.